styles/font.css

```diff
@@ -0,0 +1,21 @@
+@font-face {
+    font-family: 'Roboto';
+    font-style: normal;
+    font-weight: 400;
+    src: local(''),
```

Comment:
This is cool, that you've added local 👍 
I recommend to add woff2 format, before woff. Because woff2 compresses better

```
 src: url('font.woff2')
         format('woff2'),
         url('font.woff')
         format('woff');
```

Reply:
Done